14-01-date-input/14-1-EN-date-input.sbv

```diff
@@ -8,19 +8,19 @@ I've mentioned it in "log" and "reflog" video and in some other ones
 where I had to pass a date to Git, or specify a date output format.
 
 0:00:16.180,0:00:18.940
-First of all, lets have a look how one can set a date format.
+First of all, let's have a look how one can set a date format.
 
 0:00:18.940,0:00:20.500
-Heres a list of examples:
+Here's a list of examples:
 
 0:00:20.500,0:00:22.740
 As you might have noticed, Git can accept almost any format.
 
 0:00:23.020,0:00:25.540
-Dates precision is seconds.
+With second-accurate dates.
```

Comment:
Whats wrong with the old one?

Reply:
well if i google it i don't find any form with precision. it's fine. but fills like borrowed) as you wish, i don't make it a rule

Reply:
No-no, the second accurate is way better. What I dont like about it is the ambiguity that arises, like with first accurate dates, with second accurate dates. Will it work if we say "seconds-accurate date" or "seconds precision dates", which one is better?

Reply:
i found  one more "with precision of seconds"
hmmm "second-accurate" is not an ambiguity  is like an adjective, so accurate till last second....seconds?! let it be seconds. 1st